### PR TITLE
Improvements to `base` theme documentation

### PR DESCRIPTION
**Pull request by jenningsanderson**
Preview: 
https://dfhx9f55j8eg5.cloudfront.net/pr/67/

### CODE REVIEW

File: docusaurus/docs/reference/base/land.mdx [hunk 1]
```diff
@@ -15,6 +15,27 @@ import TabItem from '@theme/TabItem';
...
 # Land
 
+Features in the land theme come from OpenStreetMap features with the `natural` tag.
+
+<table>
+  <tr>
+    <th>Geometry Type</th>
+    <td>
+      <code>Point</code> or <code>LineString</code> or <br />
+      <code>Polygon</code> or <code>MultiPolygon</code>
```
Comment (vcschapp) [inline docusaurus/docs/reference/base/land.mdx:25]: nit: Can we do A comma B comma C comma or D and no `<br />`

File: docusaurus/docs/reference/base/landUse.mdx [hunk 2]
```diff
@@ -8,11 +8,35 @@ import generateResolverOptions from "@site/src/components/shared-libs/generateRe
...
+import LandUseExample from "!!raw-loader!@site/docs/_examples/base/landUse-example.yaml";
+import LandUseExample2 from "!!raw-loader!@site/docs/_examples/base/landUse-example.yaml";
+
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Land Use
 
+Features in the land use theme come primarily from OpenStreetMap features containing the `landuse` tag.
+
```
Comment (vcschapp) [inline docusaurus/docs/reference/base/landUse.mdx:20]: I wonder if we should adopt some kind of convention around OSM tags?

I often see them written as `landuse=*`...

File: docusaurus/docs/reference/base/landUse.mdx [hunk 3]
```diff
@@ -8,11 +8,35 @@ import generateResolverOptions from "@site/src/components/shared-libs/generateRe
...
+<table>
+  <tr>
+    <th>Geometry Type</th>
+    <td>
+      <code>Point</code> or <code>LineString</code> or <br />
+      <code>Polygon</code> or <code>MultiPolygon</code>
```
Comment (vcschapp) [inline docusaurus/docs/reference/base/landUse.mdx:26]: Commas and no line break?

**Review by vcschapp — APPROVED** [review]
Possible premature approval, as I see it's still WIP :)

**Review by AleksandraDebicka-TomTom — APPROVED** [review]

**Pull request merged**